app/src/main/java/edu/wpi/first/shuffleboard/app/WidgetPaneController.java

```diff
@@ -252,6 +260,23 @@ private void setupTile(Tile tile) {
       contextMenu.show(pane.getScene().getWindow(), event.getScreenX(), event.getScreenY());
     });
 
+    Parent parent = pane.getParent();
+    if (parent.getClass().getName().equals("com.sun.javafx.scene.control.skin.TabPaneSkin$TabContentRegion")) {
+      try {
+        // This class is package-private, so we have to use reflection to access the tab
+        Field tabField = parent.getClass().getDeclaredField("tab");
+        tabField.setAccessible(true);
+        DashboardTabPane.DashboardTab tab = (DashboardTabPane.DashboardTab) tabField.get(parent);
+        // Prefix without the protocol or leading slash
+        String namePrefix = SourceTypes.getDefault().stripProtocol(tab.getSourcePrefix()).replaceFirst("^/", "");
+        tile.getContent().allWidgets()
+            .filter(w -> w.getTitle().startsWith(namePrefix))
+            .forEach(w -> w.setTitle(w.getTitle().substring(namePrefix.length())));
+      } catch (NoSuchFieldException | IllegalAccessException e) {
+        log.log(Level.SEVERE, "Could not get the tab containing the widget pane!", e);
+      }
+    }
+
```

Comment:
from discussion: let's leave this off for now until we can refactor auto-population